lib/screens/settings/settings_screen.dart

```diff
@@ -56,6 +52,33 @@ class SettingsScreen extends StatelessWidget {
     );
   }
 
+  void _popdialog(BuildContext theContext){
+    showDialog(
+      context: theContext,
+      builder:(theContext) {
+        return AlertDialog(
+          title : Text('Log Out!'),
```

Comment:
I'd change text to plain, simple `Log out`.

Reply:
It's an alert box so an alert!!! :-D